### DISCUSSION TITLE
Hacktoberfest 2023 question

### DISCUSSION THREAD

**Discussion by Jaharmi** [netbox-community/devicetype-library#1724] [Q&A] [answered]
I know this is FOSS and everyone contributes what they can in the time they can.

I’m trying to get credit for my first Hacktoberfest with [Hacktoberfest 2023](https://hacktoberfest.com/participation/). I have a few PRs towards that goal here. This repo is tagged with the “Hacktoberfest” topic.

My PRs have passed all of the tests/checks. It looks like they would either need to be merged or tagged to get credit towards Hacktoberfest. The participation rules aren’t totally clear whether or not there's a deadline for the merging or tagging, but I assume that would need to be done before the end of the month.

Is there a chance the following PRs could be merged or tagged with the “hacktoberfest-accepted” label?

- #1679 
- #1684
- #1685

Thanks! I really do appreciate it!

**Comment by danner26** [comment]
Hey, I completely understand where you are coming from. I am the main maintainer that handles these but I have been super busy lately. I have over 100 open notifications waiting for me to work on them. I'll do my best to get around to yours ASAP!

**Reply by Jaharmi** [reply]
I think that just the `hacktoberfest-accepted` label needs to be added before the deadline, even if the PR is not merged.

Thank you for your understanding and assistance!

**Reply by danner26** [reply, marked as answer]
I added the label for you

**Reply by DanSheps** [reply]
I took a look.  The Power Connect is missing a weight parameter but I am not sure if it is available publically.  Other two are approved and just need to be merged (I will do this later)
